test-files/cmd/glacial/serial/ServiceOsmApiDbHootApiDbDcStreetsNetworkConflateTest.sh.stdout

```diff
@@ -36,12 +36,12 @@ STEP 11b: Writing a XML changeset file that is the difference between the croppe
 STEP 12: Executing the SQL changeset on the osm api db...
 
 Changeset(s) Created: 1
-Changeset Details: min_lat=38.8920931, max_lat=38.902523, min_lon=-77.0501589, max_lon=-77.0336319, num_changes=153
+Changeset Details: min_lat=38.8920931, max_lat=38.902523, min_lon=-77.0501589, max_lon=-77.0336319, num_changes=139
 Node(s) Created: 108
 Node(s) Modified: 0
 Node(s) Deleted: 0
-Way(s) Created: 20
-Way(s) Modified: 25
+Way(s) Created: 9
+Way(s) Modified: 22
```

Comment:
Less created, less modified for no reason.  That means more of the original ways were kept intact.